cmd/queue/main.go

```diff
@@ -81,11 +81,7 @@ const (
 	appResponseTimeInMsecN = "app_request_latencies"
 	queueDepthN            = "queue_depth"
 
-	// requestQueueHealthPath specifies the path for health checks for
-	// queue-proxy.
-	requestQueueHealthPath = "/health"
-
-	healthURLTemplate = "http://127.0.0.1:%d" + requestQueueHealthPath
+	healthURLTemplate = "http://127.0.0.1:%d"
```

Comment:
Is this even needed at this point?

Reply:
Sure but it's only used in one spot anyway. Not sure it's worthy of a constant.

Reply:
I prefer constant.